### PR TITLE
Translation and minor fix

### PR DESCRIPTION
**Pull request by fbsobreira**
Delete duplicate translation keys
Add missing translation for PT-BR
minor fix: replace "==" for "===", prevent warn, no cast required

### CODE REVIEW

**Comment by Rovak** [comment]
please fix the merge conflict

**Review by Jacob8765 — CHANGES_REQUESTED** [review]
Looks good, just one comment

File: src/translations/br.js [hunk 1]
```diff
@@ -166,7 +166,18 @@ export const messages = {
...
+  "welcome_to_tron": "Welcome to TRON",
+  "your_wallet": "Sua carteira",
+  "trx_balance": "Saldo TRX",
+  "remaining": "restantes",
+  "private_key": "Chave privada",
+  "show_private_key": "Exibir chave privada",
+  "register_new_account": "Criar nova conta",
+  "account_address": "endereço da conta",
+  "create_account": "Criar conta",
+  "not_started_yet": "Ainda não iniciado"
```
[the remark this answers is not in the document]
Comment (fbsobreira) [reply src/translations/br.js]: Sorry I missed that.... The merge review was so simple that I did without tests... will not happen again

**Review by Jacob8765 — APPROVED** [review]

File: src/translations/br.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jacob8765) [reply src/translations/br.js]: No worries. Thanks for fixing ;)

**Pull request closed without merging**